### PR TITLE
Index & view modified to comply w/ doc blocks.

### PR DESCRIPTION
**Pull request by sitedyno**
That code-sniffer fusses at me when I bake a new controller!

Command used:
`phpcs --colors -p --standard=vendor/cakephp/cakephp-codesniffer/CakePHP ./src ./tests`

```
----------------------------------------------------------------------
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES
----------------------------------------------------------------------
 17 | WARNING | Function return type is not void, but function has no
    |         | return statement
 34 | WARNING | Function return type is not void, but function has no
    |         | return statement
----------------------------------------------------------------------
```

### CODE REVIEW

**Comment by dereuromark** [comment]
Hm thats not ideal.
Maybe the sniffer should be relaxed here?

**Comment by markstory** [comment]
Or the docs could be Response|void.

**Comment by sitedyno** [comment]
Ok. Changing doc block instead of adding return statement.

**Comment by markstory** [comment]
Thanks!

File: src/Template/Bake/Element/Controller/index.ctp [hunk 1]
```diff
@@ -17,7 +17,7 @@
     /**
      * Index method
      *
-     * @return \Cake\Http\Response|null
+     * @return \Cake\Http\Response|void
```
Comment (dereuromark) [inline src/Template/Bake/Element/Controller/index.ctp]: Thats semantically wrong.

**Pull request merged**